apps/browser/src/background/main.background.ts

```
@@ -506,6 +507,8 @@ export default class MainBackground {
       this.userVerificationApiService
     );
 
+    this.configApiService = new ConfigApiService(this.apiService, this.authService);
```

I'm a bit concerned about this addition. The `ConfigService` instance within the extension background originally pointed to an abstract class of `ConfigApiService`. As a result, calls to the service present within the extension background were silently failing and being ignored.

Since we need to check `/config` for the `autofill-v2` feature flag, we need to ensure that an instantiated class of `ConfigApiService` is present. 

However, I was noticing that we make SEVERAL calls to the config when initializing the extension... This likely isn't something we want to do, and I'm wondering if it's worth exploring implementing a cached/stored value within the `ConfigService`.

@MGibson1 Can I please ask for your opinion on this? Just want to make sure we're not hammering the API unnecessarily. That said, if this isn't a huge concern then I'll hold off on caching the config response.